packages/docusaurus-init/templates/classic/docs/deploy-your-site.md

```diff
@@ -1,25 +1,20 @@
 ---
-title: Deploy your site 
+title: Deploy your site
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
-This page will discuss multiple options available when it comes to 
-deploying your docs site. Before even this happens, you need to build the files 
-of your website for production. To do this, run :
+This page will discuss multiple options available when it comes to deploying your docs site. Before even this happens, you need to build the files of your website for production. To do this, run :
 
 ```bash
 npm run build
 ```
+
 The static files will be generate in the build/ directory.
 
 ## Self Hosting
 
-:::warning
-It is not the most performant solution
-:::
-
+:::warning It is not the most performant solution :::
```

Comment:
unfortunately this code is not valid anymore after prettier has been run 😅 that's why I said there were missing line breaks, because you have to add extra line breaks in some places before running prettier

Reply:
going to revert your commit and do the fixes directly so I can merge asap